index.html

```diff
@@ -5,9 +5,32 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width">
     <title>Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;700;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+    <header class="header">
+        <a href="#" class="logo">My Logo</a>
+        <img src="./img/icons/menu.png" alt="menu" class="mobile-menu">
+    </header>
+
+    <section class="headline">
+        <h1>
+            I’m Tolibjon Glad to see you!
+        </h1>
+
+        <p>
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <a href="#" class="connect-me">LET'S CONNECT</a>
+        <div class="social-media">
+            <a href="https://twitter.com/tolib_tolibov"><img src="./img/icons/twitter.svg" alt="Twitter"></a>
+            <a href="https://www.linkedin.com/in/tolibjon-tolibov/"><img src="./img/icons/linkedin.svg" alt="Linkedin"></a>
+            <a href="#"><img src="./img/icons/medium.svg" alt="Medium"></a>
+            <a href="https://github.com/toliboff"><img src="./img/icons/github.svg" alt="GitHub"></a>
+            <a href="#"><img src="./img/icons/victory.svg" alt="Victory"></a>
+        </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Kindly remove the `div` and use the proper semantic `ul` tag for this section. Also, as suggested above, consider using [Font Awesome](https://fontawesome.com/v4.7/icons/) for the icons.

Reply:
:+1: This issue has been addressed correctly :heavy_check_mark: